src/assets/images/syncrobit.svg

```diff
@@ -0,0 +1 @@
88,60.88,0,0,0-1.36,6.23,5.36,5.36,0,0,0-.06.69,4,4,0,0,1-.06.72,1,1,0,0,1-.15.4h-.06A.57.57,0,0,1,46.19,45.49ZM40.46,39.3a9.13,9.13,0,0,1,1.11-1.19A25.93,25.93,0,0,1,45.93,35a27.67,27.67,0,0,1,5-2.27,9.46,9.46,0,0,1,2.42-.51l.42,0h-.24l-.18,0a7.49,7.49,0,0,0-2,.39A26,26,0,0,0,45.85,35a23.34,23.34,0,0,0-4.65,3.4,6.71,6.71,0,0,0-.75.84l-.06.13Zm9.08,5.22a10,10,0,0,0,2.77-2.06,12.44,12.44,0,0,1-1.79,1.4,13.35,13.35,0,0,1-1.9,1,3.53,3.53,0,0,1-.42.18l-.06,0h.07A5.82,5.82,0,0,0,49.54,44.52Zm-.78-4c.11,0,.23-.08.38-.15a9.84,9.84,0,0,1,2.34-.66l.35-.06h-.25a8,8,0,0,0-2.43.65,1.92,1.92,0,0,0-.57.27h0Zm.78,3.06a3.86,3.86,0,0,0,.66-.4h0a3.1,3.1,0,0,0-.53.24l-.41.23-.39.24a.71.71,0,0,0-.16.13A5.47,5.47,0,0,0,49.54,43.6Z" transform="translate(-0.5 -0.5)"/><path d="M54.71,43.52c0-.32.15-.49.46-.49a.68.68,0,0,1,.28.07.75.75,0,0,1,.35.53.54.54,0,0,1-.23.4.79.79,0,0,1-.54.25C54.81,44.28,54.71,44,54.71,43.52Z" transform="translate(-0.5 -0.5)"/><path d="M56.48,43.51a1.06,1.06,0,0,1-.31-.36,1.28,1.28,0,0,1-.08-.47,4.33,4.33,0,0,1,.22-1.2c.15-.48.3-.73.45-.73a.5.5,0,0,1,.2.09q.09.09.09.12a3.33,3.33,0,0,1-.07.46,10.92,10.92,0,0,0-.12,1.46c0,.12,0,.19,0,.19a2.65,2.65,0,0,0,.5-.28,7.89,7.89,0,0,0,1.23-1,7.49,7.49,0,0,0,.7-.72c.14-.19.27-.28.37-.28a.14.14,0,0,1,.14.09.35.35,0,0,1,0,.2c0,.25-.41.8-1.23,1.64-.65.65-1.15,1-1.5,1A1,1,0,0,1,56.48,43.51ZM56.55,40a.5.5,0,0,1-.15-.33v-.12a1.67,1.67,0,0,1,.35-.73c.19-.25.36-.38.52-.38a.38.38,0,0,1,.25.13c.07.08.1.13.1.18s0,.06,0,.06a.2.2,0,0,1-.11,0s0,0,0,0a0,0,0,0,0,0,0s0,0,0,0a.11.11,0,0,0,0,.08.72.72,0,0,1,.18.49.92.92,0,0,1-.21.58.63.63,0,0,1-.44.25A.59.59,0,0,1,56.55,40Zm.56-.7a.55.55,0,0,0,.06-.19s0,0,0,0L57,39.3c-.06.14-.09.21-.09.22h0S57,39.45,57.11,39.3Zm1,3.66A10,10,0,0,0,59.1,42a3.58,3.58,0,0,0,.37-.44l-.08.08c-.38.42-.7.76-1,1a3.71,3.71,0,0,0-.34.31Z" transform="translate(-0.5 -0.5)"/><path d="M59.75,45.47a5,5,0,0,1-.28-1.28c0-.27,0-.81,0-1.61a21.65,21.65,0,0,1,.41-4.46,1.68,1.68,0,0,0,.06-.31h0l-.29,0L59,38c-.21.07-.38.14-.52.2s-.42.21-.42.27v0s0,0,0,0a.21.21,0,0,1-.14-.07c-.07-.06-.11-.11-.11-.16a.23.23,0,0,1,0-.09c0-.17.29-.36.74-.57a6.76,6.76,0,0,1,1.2-.41l.35-.07a2.69,2.69,0,0,1,.15-.49,12.3,12.3,0,0,1,1.32-3h.05a.47.47,0,0,1,.35.17.05.05,0,0,1,0,0,.15.15,0,0,1,0,.07,2.25,2.25,0,0,1-.11.37A24.15,24.15,0,0,0,61,37.07a6.42,6.42,0,0,0,.77,0,9.57,9.57,0,0,1,1.12.07q.9.14.9.57c0,.07,0,.11,0,.11a.52.52,0,0,1-.18-.06,2,2,0,0,0-.57-.14c-.23,0-.56,0-1,0s-.79,0-1.14,0a24.32,24.32,0,0,0-.69,5.62,6.6,6.6,0,0,0,.31,2.49l.1.22a.14.14,0,0,1-.13.05A.78.78,0,0,1,59.75,45.47ZM60.58,38c0-.18.06-.29.06-.31h0a5.69,5.69,0,0,0-.18.79A26,26,0,0,0,60,41.81c0,.38,0,.68,0,.89,0,.54,0,.8,0,.79A26.89,26.89,0,0,1,60.58,38ZM60,44.12V44a.55.55,0,0,0,0,.12Zm0,.29h0v.06S60,44.45,60,44.41Zm.9-7.72c.18-.64.39-1.23.61-1.81a2.6,2.6,0,0,0,.14-.37A13.07,13.07,0,0,0,61,36.17c-.16.48-.24.78-.27.9a.06.06,0,0,0,0,0A1.81,1.81,0,0,0,60.93,36.69Zm.78-2.25s0,0,0,0a.06.06,0,0,0,0,0l-.06.08h0A0,0,0,0,0,61.71,34.44Z" transform="translate(-0.5 -0.5)"/></g></svg>
```

Comment:
Something is up with this SVG it seems to work with:

```
<svg width="75" height="75" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#clip0)"><path d="M14 1h46.9a13 13 0 0113 13v46.9a13 13 0 01-13 13H14a13 13 0 01-13-13V14A13 13 0 0114 1v0z" stroke="currentColor" stroke-width="2"/><path d="M16.7 7.2h41.5a9.56 9.56 0 019.5 9.5v41.5a9.56 9.56 0 01-9.5 9.5H16.7a9.56 9.56 0 01-9.5-9.5V16.7a9.5 9.5 0 019.5-9.5v0z" stroke="currentColor"/><path d="M15.5 45.63a.31.31 0 01-.15-.25c0-.25.3-.68.89-1.28l.27-.29a.751.751 0 00.17-.27c.1-.307.154-.627.16-.95a.74.74 0 00-.07-.37c-.1-.15-.43-.26-1-.32-.483-.03-.967-.03-1.45 0-.471.004-.942-.019-1.41-.07a2.001 2.001 0 01-.72-.27 1.48 1.48 0 01-.69-1.31 1.35 1.35 0 010-.33 9.12 9.12 0 012-3.61 17.419 17.419 0 013.83-3.42 6.63 6.63 0 013.17-1.18c.19-.006.38.035.55.12.244.11.452.287.6.51.119.225.18.476.18.73a3 3 0 01-.59 1.48 36.287 36.287 0 01-2.63 3.21c-.59.68-1.31 1.52-2.15 2.53-.09.11-.16.17-.2.17a.72.72 0 01-.32-.13.38.38 0 01-.16-.29 21.51 21.51 0 012-2.33c1.25-1.38 2.08-2.35 2.49-2.9a4.51 4.51 0 001.09-2.25s0-.07-.06-.07a.999.999 0 00-.5-.09c-.22.006-.437.036-.65.09a9.84 9.84 0 00-3.79 2.15c-.23.2-.52.48-.88.84a14.897 14.897 0 00-2.57 3.31 4.39 4.39 0 00-.7 2 1.113 1.113 0 000 .2c0 .1.09.17.22.22.13.05.54 0 1.32.07.587 0 1.027.02 1.32.06a2.83 2.83 0 011.92.68c.25.245.415.564.47.91v.13l.44-.29a2.829 2.829 0 011-.52.37.37 0 01.17 0 .26.26 0 01.12.11.259.259 0 010 .1 6.86 6.86 0 00-1 .68c-.354.22-.67.493-.94.81a3.89 3.89 0 01-.75 1.37 1.08 1.08 0 01-.68.43.47.47 0 01-.32-.12zm-3-6.33a13.867 13.867 0 012.1-3.09 15.42 15.42 0 012.86-2.57 8.686 8.686 0 012.2-1.14l.25-.11-.32.1a11.6 11.6 0 00-4.27 2.89 14.24 14.24 0 00-2.82 3.86c-.05.097-.093.197-.13.3 0 0 .02-.04.13-.24zm3.68 6.13c.187-.153.352-.331.49-.53a1.92 1.92 0 00.4-.67 1.99 1.99 0 00-.14.23c-.227.36-.492.696-.79 1l.04-.03zm2.79-8.23c.095-.093.182-.193.26-.3a3.191 3.191 0 00-.27.29l-.2.21-.08.1.29-.3zm.46-.52a1.816 1.816 0 00-.19.18l.19-.18zm.16-.19c-.12.1-.17.15-.17.17a.72.72 0 00.15-.16l.02-.01zm.1-.1a.17.17 0 01.06-.08.598.598 0 00-.13.11l.07-.03zm.17-.22c-.05 0-.08.06-.08.07l.08-.07zm.26-3.84h-.07s.05.01.05 0h.02z" fill="currentColor"/><path d="M21.28 47.2a.668.668 0 01-.22-.07 1.1 1.1 0 01-.6-1c.01-.375.103-.744.27-1.08a3.22 3.22 0 01.41-.7c.207-.26.43-.508.67-.74a7.49 7.49 0 011.19-1l.24-.17s.08-.44.18-1.18c.019-.356.019-.713 0-1.07 0-.68 0-1-.1-1a10.79 10.79 0 00-1.55 2.15c-.7 1-1.23 1.58-1.57 1.58a.998.998 0 01-.75-.28 1 1 0 01-.27-.76c.038-.473.146-.938.32-1.38.14-.5.3-.76.47-.76a.51.51 0 01.32.17c-.12.62-.19 1-.21 1.16-.04.305-.06.612-.06.92v.2l.13-.11c.5-.557.952-1.156 1.35-1.79a11 11 0 011-1.34.818.818 0 01.58-.28 1 1 0 01.65.3c.22.259.35.581.37.92v.75c.005.963-.103 1.922-.32 2.86-.57 2.46-1.33 3.7-2.28 3.7h-.22zm.8-6.62c.46-.69.8-1.17 1-1.47l-.17.18c-.364.437-.698.898-1 1.38a20.24 20.24 0 01-1.2 1.67 7.41 7.41 0 001.37-1.76zm-.42 6.13c.17-.19.317-.397.44-.62.219-.42.4-.858.54-1.31.187-.529.337-1.07.45-1.62.022-.082.039-.166.05-.25a.77.77 0 00-.19.15c-.29.26-.48.44-.58.55a4.901 4.901 0 00-1.21 1.89c-.062.18-.1.37-.11.56v.19c.008.168.052.331.13.48.05.13.11.19.17.19.06 0 .15-.07.31-.21zm1.2-.92a5.48 5.48 0 00.34-.81c.179-.528.33-1.066.45-1.61.11-.58.17-.9.17-.93l-.12.53a12.883 12.883 0 01-.8 2.72 1.155 1.155 0 00-.06.17l.02-.07zm2.53-2.81a.72.72 0 01-.36-.41c-.019-.15-.019-.3 0-.45a3.67 3.67 0 010-.58c.083-.62.24-1.228.47-1.81.08-.17.15-.26.22-.26h.07a.28.28 0 01.21.12.34.34 0 01.1.22c0 .13 0 .34-.07.65-.11.72-.17 1.26-.19 1.65.13-.14.27-.34.46-.59.224-.355.478-.69.76-1a.49.49 0 01.34-.15c.3 0 .47.25.53.76.06.51.1.55.14.55a1.259 1.259 0 00.55-.27c.16-.13.34-.27.51-.43.17-.16.23-.19.31-.19.08 0 .12.05.12.15 0 .1-.24.42-.72.88a1.831 1.831 0 01-.36.29.467.467 0 01-.16 0 .75.75 0 01-.42-.14 1.089 1.089 0 01-.47-.89v-.19a5.08 5.08 0 00-.8 1.26c-.129.281-.28.552-.45.81a.34.34 0 01-.29.13.89.89 0 01-.5-.11zm.85-.48c.094-.146.174-.3.24-.46l-.47.76a.273.273 0 000 .07c.088-.116.165-.24.23-.37z" fill="currentColor"/><path d="M29.32 43.01a1.32 1.32 0 01-.36-.41 1.084 1.084 0 010-.28 4.074 4.074 0 01.37-1.62 2.13 2.13 0 01.68-.93.419.419 0 01.21-.06.46.46 0 01.32.18 1.13 1.13 0 01.27.84v.22a3.51 3.51 0 01-.05.41c0 .16-.07.24-.11.24h-.07a.28.28 0 01-.14-.12.34.34 0 01-.09-.25 3.649 3.649 0 010-.38c.014-.16.014-.32 0-.48v-.14s-.1.09-.22.27a3.695 3.695 0 00-.36.81 4.427 4.427 0 00-.27 1.33v.09c.186-.022.366-.08.53-.17.69-.4 1.329-.883 1.9-1.44.42-.36.65-.57.68-.6l.15-.11h.06a.31.31 0 01.13.1c.033.05.05.11.05.17-.247.374-.547.711-.89 1-.42.42-.76.73-1 .95a2.16 2.16 0 01-1.27.6 1 1 0 01-.52-.22zm1.24-.14h.06-.1.04zm.16-.09c.268-.168.517-.366.74-.59l.25-.24c-.29.25-.55.47-.79.65l-.27.21.07-.03z" fill="currentColor"/><path d="M33.9 43.09c-.27-.08-.41-.51-.41-1.28a4.991 4.991 0 00-.08-.83h-.12a1.393 1.393 0 01-.32-.05 1.05 1.05 0 01-.76-.53 1.001 1.001 0 01-.05-.34c.027-.503.142-.997.34-1.46a2.56 2.56 0 01.7-1.05.7.7 0 01.38-.11c.293.004.573.122.78.33a.471.471 0 01.17.37c-.077.428-.226.84-.44 1.22-.29.61-.55 1-.77 1-.05 0-.08 0-.08.06s.05 0 .15 0a1 1 0 01.82.45c.016.123.016.247 0 .37v1.32c.08 0 .23-.13.46-.37.326-.32.605-.683.83-1.08.06-.2.13-.29.2-.29a.16.16 0 01.09 0s.05.06.05.09a3.2 3.2 0 01-.42 1.05 5.262 5.262 0 01-.78.92.75.75 0 01-.51.19.67.67 0 01-.23.02zm-.85-3.19c.073-.39.194-.77.36-1.13.07-.125.12-.26.15-.4a.85.85 0 00-.11.21 3.54 3.54 0 00-.48 1.48.601.601 0 000 .23l.08-.39zm.81-.42a.387.387 0 000-.09 5.32 5.32 0 00.3-.69 2.17 2.17 0 00.13-.48v-.08a1.797 1.797 0 00-.07.22c-.133.4-.273.79-.42 1.17 0 0 .05-.03.06-.03v-.02zm.25 3.09v-.84a1.749 1.749 0 00-.06-.67v.11c-.01.494.007.988.05 1.48l.01-.08z" fill="currentColor"/><path d="M35.88 42.85a1.2 1.2 0 01-.53-1 2.44 2.44 0 01.48-1.35 2 2 0 01.63-.64.189.189 0 01.1 0 .13.13 0 01.07 0l.14.07c.059.078.11.162.15.25.539.016 1.076.062 1.61.14a.52.52 0 01.46.32c.01.04.01.08 0 .12-.189.03-.381.03-.57 0h-.82c-.245.001-.49.025-.73.07.222.22.387.492.48.79.008.083.008.167 0 .25a.74.74 0 01-.07.31 1.341 1.341 0 01-.27.33 1.102 1.102 0 01-.77.41.78.78 0 01-.36-.07zm.47-.75c.108-.062.209-.136.3-.22.07-.071.131-.152.18-.24a.41.41 0 00.09-.2l-.08-.1-.19-.19-.23-.23c-.14.242-.257.497-.35.76-.065.17-.105.349-.12.53a.901.901 0 00.4-.11zm.3.6a1 1 0 00.31-.23c.12-.12.19-.19.19-.22s-.07 0-.17.14l-.2.17a.952.952 0 01-.17.11l-.11.07.15-.04zm9.04 2.29a.66.66 0 01-.23-.44c0-.17 0-.45.09-.84.035-.181.055-.365.06-.55.013-.12.013-.24 0-.36a3.45 3.45 0 00-3.64-3c-.23 0-.62 0-1.14.06h-.37a1.23 1.23 0 01-1.13-.59.83.83 0 01-.12-.4c0-.25.29-.66.88-1.22a23.59 23.59 0 013.61-2.74 34.828 34.828 0 014.57-2.46 13.118 13.118 0 014.6-1.34c.186-.013.374-.013.56 0 .19.04.36.146.48.3l.17.22c.129.202.188.441.17.68a3.53 3.53 0 01-.85 2 35.601 35.601 0 01-3.36 3.79c-.51.52-.77.79-.77.8h.09a6.587 6.587 0 011.91-.35 1.78 1.78 0 01.44 0c.142.024.28.068.41.13.126.08.24.178.34.29.102.107.193.224.27.35a1 1 0 01.13.35c.017.16.017.32 0 .48.013.16.013.32 0 .48-.273 1.013-1.273 2.06-3 3.14a5.471 5.471 0 01-2.35 1h-.12a.52.52 0 01-.22-.1.75.75 0 01-.29-.34.64.64 0 01-.07-.28c0-.07.07-.17.22-.32a3.08 3.08 0 01.43-.37c.456-.346.95-.641 1.47-.88a3.138 3.138 0 011.16-.33.548.548 0 01.25.05v.06l.12-.08.2-.16c.285-.2.546-.432.78-.69.225-.206.43-.434.61-.68a4 4 0 00.29-.47c.071-.159.119-.327.14-.5l-.14-.1a1.67 1.67 0 00-.95-.24 8.347 8.347 0 00-2.81.73 2.84 2.84 0 01-.56.21.38.38 0 01-.29-.25 1.25 1.25 0 01-.14-.41.58.58 0 01.14-.34c.41-.53 1.54-1.76 3.4-3.67.62-.64 1.1-1.16 1.46-1.57a7 7 0 001.3-2l-.06-.06c-.11-.12-.43-.18-.94-.18a6.258 6.258 0 00-1.71.28 25.996 25.996 0 00-6.1 2.76 18.799 18.799 0 00-4.7 3.74c-.078.087-.148.18-.21.28h1.15a5.3 5.3 0 012.34.57 5.13 5.13 0 012.13 2.25l.15.25.24-1c.79-3.38 1.41-5.35 1.86-5.93.1-.15.18-.23.23-.23a.249.249 0 01.11 0 .27.27 0 01.1.11.2.2 0 010 .14v.07a8.998 8.998 0 00-.66 1.91 60.866 60.866 0 00-1.36 6.23 5.35 5.35 0 00-.06.69c.002.241-.018.482-.06.72-.02.143-.072.28-.15.4h-.06a.57.57 0 01-.44-.05zm-5.73-6.19a9.118 9.118 0 011.11-1.19 25.928 25.928 0 014.36-3.11 27.67 27.67 0 015-2.27 9.456 9.456 0 012.42-.51h.42-.42a7.487 7.487 0 00-2 .39 25.993 25.993 0 00-5.5 2.39 23.333 23.333 0 00-4.65 3.4c-.273.259-.524.54-.75.84l-.06.13.07-.07zm9.08 5.22a10.001 10.001 0 002.77-2.06c-.553.52-1.152.989-1.79 1.4-.606.384-1.24.718-1.9 1-.136.07-.276.13-.42.18h-.06.07c.463-.118.91-.293 1.33-.52zm-.78-4c.11 0 .23-.08.38-.15a9.838 9.838 0 012.34-.66l.35-.06h-.25c-.84.087-1.66.307-2.43.65-.204.058-.396.15-.57.27l.18-.05zm.78 3.06c.233-.11.454-.245.66-.4a3.093 3.093 0 00-.53.24l-.41.23-.39.24a.708.708 0 00-.16.13c.288-.116.566-.257.83-.42v-.02zm5.17-.06c0-.32.15-.49.46-.49a.68.68 0 01.28.07.75.75 0 01.35.53.541.541 0 01-.23.4.79.79 0 01-.54.25c-.22 0-.32-.28-.32-.76zm1.77-.01a1.059 1.059 0 01-.31-.36 1.28 1.28 0 01-.08-.47c.016-.408.09-.812.22-1.2.15-.48.3-.73.45-.73.073.015.14.046.2.09.06.06.09.1.09.12a3.342 3.342 0 01-.07.46c-.073.483-.113.971-.12 1.46v.19c.176-.075.344-.169.5-.28a7.897 7.897 0 001.23-1c.25-.224.483-.465.7-.72.14-.19.27-.28.37-.28a.14.14 0 01.14.09.35.35 0 010 .2c0 .25-.41.8-1.23 1.64-.65.65-1.15 1-1.5 1a1 1 0 01-.59-.21zm.07-3.51a.5.5 0 01-.15-.33v-.12c.055-.268.175-.519.35-.73.19-.25.36-.38.52-.38a.38.38 0 01.25.13c.07.08.1.13.1.18v.06a.202.202 0 01-.11 0 .11.11 0 000 .08.72.72 0 01.18.49.92.92 0 01-.21.58.63.63 0 01-.44.25.591.591 0 01-.49-.21zm.56-.7a.551.551 0 00.06-.19l-.17.19c-.06.14-.09.21-.09.22 0 .01.09-.07.2-.22zm1 3.66c.352-.297.682-.617.99-.96.135-.136.259-.284.37-.44l-.08.08c-.38.42-.7.76-1 1-.12.096-.233.2-.34.31l.06.01z" fill="currentColor"/><path d="M58.97 43.69a5 5 0 00.28 1.28.78.78 0 00.74.53.14.14 0 00.13-.05l-.1-.22a6.599 6.599 0 01-.31-2.49c.01-1.894.242-3.78.69-5.62h2.14c.196.019.388.066.57.14a.52.52 0 00.18.06v-.11c0-.287-.3-.477-.9-.57a9.574 9.574 0 00-1.12-.07 6.422 6.422 0 01-.77 0c.228-.967.515-1.919.86-2.85.047-.12.084-.244.11-.37a.15.15 0 000-.07.469.469 0 00-.35-.17h-.05a12.298 12.298 0 00-1.32 3 2.69 2.69 0 00-.15.49l-.35.07a6.76 6.76 0 00-1.2.41c-.45.21-.74.4-.74.57a.231.231 0 000 .09c0 .05.04.1.11.16a.21.21 0 00.14.07c0-.06.28-.21.42-.27.14-.06.31-.13.52-.2l.65-.19h.29c-.01.105-.03.209-.06.31a21.649 21.649 0 00-.41 4.46v1.61z" fill="currentColor"/></g><defs><clipPath id="clip0"><path fill="none" d="M0 0h74.9v74.9H0z"/></clipPath></defs></svg>
```

Reply:
Replaced with the suggested one